README.md

```diff
@@ -168,3 +168,23 @@ Additionally, response can contain additional response body. All fields of the r
   "payment": {"id":"XXX", "intent":"sale", ...}  # only in case of getting the payment object 
 }
 ```
+
+## Possible edge cases
+1. First case:
+    1. user inputs an address Berlin, Germany
+    1. he clicks on Continue and is redirected to Paypal payment page with this address.
+    1. In a different tab, the same user changes his address to Paris, France.
+    1. The user confirms the payment in the first tab
+    1. Paypal has the address of Berlin, but the shop will deliver the goods to Paris.
+    
+    **Possible solution:** the backend calls `patch/payments` endpoint every time user changes it.
+
+1. Second case:
+    1. user has cart with item1=10€
+    1. user is redirected to Paypal payment page
+    1. in a different tab, user changes his cart to e.g. item2=20€
+    1. user completes the payment in the first tab
```

Comment:
_and redirects back to approval page_

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/82/commits/865ce6bd9ab6a2fbff99581125d80a0dc0c9cdbd